infra/base-images/base-builder/compile

```diff
@@ -186,6 +200,25 @@ else
   fi
 fi
 
+if [ "$SANITIZER" = "introspector" ]; then
+  unset CXXFLAGS
+  unset CFLAGS
+  apt-get install -y libjpeg-dev zlib1g-dev
+  pip3 install --upgrade setuptools
+  pip3 install cxxfilt pyyaml beautifulsoup4 lxml soupsieve matplotlib
+  mkdir $SRC/inspector-tmp
```

Comment:
nit: Just call it inspector, not inspector-tmp

Reply:
Done.